modules/lsp/src/main/scala/playground/lsp/ModelLoader.scala

```diff
@@ -56,28 +58,12 @@ object ModelLoader {
   ): Model = Model
     .assembler()
     .putProperty(ModelAssembler.DISABLE_JAR_CACHE, true)
-    .pipe(addModelsFromJars(classLoader.getURLs().map(_.toURI()).map(Paths.get(_).toFile())))
+    .discoverModels(classLoader)
```

Comment:
This only ever gets used with the isolated classloader, so should be equivalent to what was happening before. Tests seem to validate that.